todo_app/templates/itemForm.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+<div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+	    <h2>Add a new item</h2>
+	    <form action="http://localhost:5000/addItem" method="post">
```

Comment:
This URL will not work on other servers.

You should change it to just `action="/addItem"` to use a relative path, or you can use url_for:
`action="{{url_for('add_item')}}`